src/sharedServices/sns/publishToSNS.ts

```diff
@@ -0,0 +1,13 @@
+import { SNSClient, PublishCommand } from '@aws-sdk/client-sns'
```

Comment:
Can we add some tests for this? Apart from anything else, it'll push our test coverage down if we don't, and also every other similar bit of code that interacts with the AWS SDK currently has tests.

Reply:
added a test